app/src/main/java/com/duckduckgo/app/location/ui/LocationPermissionsActivity.kt

```diff
@@ -98,7 +98,7 @@ class LocationPermissionsActivity : DuckDuckGoActivity(), SiteLocationPermission
     }
 
     private fun editSiteLocationPermission(entity: LocationPermissionEntity) {
-        val dialog = SiteLocationPermissionDialog.instance(entity.domain, true)
+        val dialog = SiteLocationPermissionDialog.instance(entity.domain, true, "")
```

Comment:
can we use named params here? I have no idea what that extra string param means without drilling into the method

Reply:
Done!